Fail with panic after importing AWS ALB target groups

**General information:**

* Operating System: macOS Big Sur 11.5.1
* Terracognita version / tag: 0.7.1 (release binary & docker image) & 0.6.4 (from homebrew)
* Did you build Terracognita from sources or did you use the Docker image: I tried the release binary, docker image and homebrew installation.

**Describe the bug:**

When I try to do a first import from AWS it is busy for a while making progress with different resources but then fails with `panic: runtime error: invalid memory address or nil pointer dereference` when importing the ALB target groups.

**Log message**
```
Starting Terracognita with version v0.7.1
Importing with filters:
	Tags:    [],
	Include: [],
	Exclude: [],
	Targets: [],
Importing aws_instance [144/144] Done!
Importing aws_alb [37/37] Done!
Importing aws_alb_listener [67/67] Done!
Importing aws_alb_listener_certificate [117/117] Done!
Importing aws_alb_listener_rule [149/149] Done!
Importing aws_alb_target_group [103/103] Done!
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x5f6531f]

goroutine 1 [running]:
github.com/cycloidio/terracognita/aws.albTargetGroupAttachments(0xfbe08d8, 0xc00012e008, 0xc00054a380, 0xec6e851, 0x1f, 0xc0016d9960, 0xc0009176a0, 0xc002faa600, 0xa, 0x10, ...)
	/tmp/build/6b18b934/code/aws/resources.go:476 +0x1ff
github.com/cycloidio/terracognita/aws.(*aws).Resources(0xc00054a380, 0xfbe08d8, 0xc00012e008, 0xec6e851, 0x1f, 0xc0016d9960, 0xc002c6fe00, 0x30, 0x0, 0x0, ...)
	/tmp/build/6b18b934/code/aws/provider.go:86 +0xea
github.com/cycloidio/terracognita/provider.Import(0xfbe08d8, 0xc00012e008, 0xfc6a608, 0xc00054a380, 0xfbe1f98, 0xc00054a400, 0x0, 0x0, 0xc0016d9960, 0xfaaba80, ...)
	/tmp/build/6b18b934/code/provider/import.go:91 +0x1d3d
github.com/cycloidio/terracognita/cmd.glob..func2(0x16a7e400, 0xc000141080, 0x0, 0x8, 0x0, 0x0)
	/tmp/build/6b18b934/code/cmd/aws.go:110 +0x89e
github.com/spf13/cobra.(*Command).execute(0x16a7e400, 0xc000141000, 0x8, 0x8, 0x16a7e400, 0xc000141000)
	/tmp/build/6b18b934/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:842 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0x16a7ebe0, 0xc0000a4778, 0xc000467f78, 0x1006045)
	/tmp/build/6b18b934/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/tmp/build/6b18b934/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
main.main()
	/tmp/build/6b18b934/code/main.go:11 +0x31
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -7,6 +7,11 @@
 - Resource names now are generated removing invalid characters instead just assigning a random alphanumeric value
   ([Issue #208](https://github.com/cycloidio/terracognita/issues/208))
 
+### Fixed
+
+- Import with `aws_alb_target_group_attachment` now validates if the needed values are present
+  ([Issue #213](https://github.com/cycloidio/terracognita/issues/213))
+
 ## [0.7.1] _2021-07-15_
 
 ### Added

--- a/aws/resources.go
+++ b/aws/resources.go
@@ -478,6 +478,11 @@ func albTargetGroupAttachments(ctx context.Context, a *aws, resourceType string,
 		}
 
 		for _, t := range targetHealths {
+			// As this are the required values to get the resource
+			// we validate that they are present
+			if t.Target == nil || i.TargetGroupArn == nil {
+				continue
+			}
 			r, err := initializeResource(a, fmt.Sprintf("%s_%d_%s", *t.Target.Id, *t.Target.Port, *i.TargetGroupArn), resourceType)
 			if err != nil {
 				return nil, err

